index.js

```diff
@@ -809,22 +810,26 @@ class Tilesets {
 
   _addFiles(tilesetId, readCb) {
     const path = `tileset/${TILESET_NAMES[tilesetId]}`
-    const promises = ['.cv5', '.vx4', '.vr4', '.wpe'].map(extension => readCb(path + extension))
+    const isExtended = fs.existsSync(
```

Comment:
This will probably be fixed with the above change, but definitely avoid the `Sync` versions of the file functions here. You can chain promises to get the behavior you want